pkg/apis/meta.go

```diff
@@ -251,16 +196,11 @@ func (m *grafanaResourceMetaAccessor) SetOriginInfo(info *ResourceOriginInfo) {
 			anno[AnnoKeyOriginTimestamp] = info.Timestamp.Format(time.RFC3339)
 		}
 	}
-	acc.SetAnnotations(anno)
+	m.obj.SetAnnotations(anno)
 }
 
 func (m *grafanaResourceMetaAccessor) GetOriginInfo() (*ResourceOriginInfo, error) {
-	acc, err := meta.Accessor(m.obj)
-	if err != nil {
-		return nil, err
-	}
-
-	v, ok := acc.GetAnnotations()[AnnoKeyOriginName]
+	v, ok := m.obj.GetAnnotations()[AnnoKeyOriginName]
```

Comment:
@DanCech -- any reason to parse this every time?

Reply:
not sure I understand the question